content/docs/command-reference/run.md

```diff
@@ -227,14 +227,13 @@ $ dvc run -n my_stage './my_script.sh $MYENVVAR'
   It's used by `dvc repro` to change the working directory before executing the
   `command`.
 
-- `--no-exec` - create a stage file, but do not execute the `command` defined in
-  it, nor cache dependencies or outputs (like with `--no-commit`, explained
-  below). DVC will also add your outputs to `.gitignore`, same as it would do
-  without `--no-exec`. Use `dvc commit` to force committing existing output file
-  versions to cache.
-
-  This is useful if, for example, you need to build a pipeline quickly first,
-  and run it all at once later.
+- `--no-exec` - create a stage, but do not execute the `command` defined in it.
+  DVC will still add the outputs to `.gitignore`. If the outputs already exist
+  in the workspace, use `dvc commit` to force DVC to track them (as if the stage
+  had run).
+
+  This is useful, for example, to define a full `dvc.yaml` pipeline first, and
```

Comment:
This in this context reads as it belongs to the existing outputs ... it's confusing a bit

Reply:
Hmmmm it's a separate paragraph so it shouldn't read like that. But I added "This option" at the beginning just in case.